frontend/src/components/editor/actions/useNotebookActions.tsx

```diff
@@ -375,6 +383,16 @@ export function useNotebookActions() {
       label: "Hide all markdown code",
       handle: hideAllMarkdownCode,
     },
+    {
+      icon: <ChevronRightCircleIcon size={14} strokeWidth={1.5} />,
+      label: "Collapse all columns",
```

Comment:
can this be `Collapse/Expand all sections`? 

and `column` →  `section` in other places